main.go

```diff
@@ -820,7 +813,10 @@ to get your bearings.
 		}
 
 		v := connect()
-		if opt.Move.Recurse {
+
+		//Don't try to recurse if operating on a key
+		// args[0] is the source path. args[1] is the destination path.
+		if opt.Move.Recurse && !vault.PathHasKey(args[0]) && !vault.PathHasKey(args[1]) {
```

Comment:
Does this mean we can't do `safe cp -R secret/path/to/thing:key secret/path/to/new/thing` or `safe cp -R secret/path/to/thing:key secret/path/to/new/thing:newkey`?

Sure, the `-R` doesn't make too much sense for those cases, but if it's being scripted, and the user is supplying the paths, it could be useful to support `-R` with things that dont need recursion